pulp_ansible/app/viewsets.py

```diff
@@ -173,6 +176,14 @@ class Meta:
         fields = ["namespace", "name", "version", "q", "is_highest", "tags"]
 
 
+class CollectionRemoteFilter(RemoteFilter):
+    url = filters.CharFilter(field_name="url")
+
+    class Meta:
+        model = CollectionRemote
+        fields = ["url"]
```

Comment:
Can you verify that we keep the filters from the superclass when we specify this?

Reply:
hmm... so when I added the Meta class to the `CollectionRemoteFilter`, filters are missing inherited fields:
```
['url', 'pulp_label_select', 'ordering']
```

and without Meta class:
```
['name', 'name__in', 'name__icontains', 'name__contains', 'name__startswith', 'pulp_last_updated', 'pulp_last_updated__lt', 'pulp_last_updated__lte', 'pulp_last_updated__gt', 'pulp_last_updated__gte', 'pulp_last_updated__range', 'pulp_label_select', 'url', 'ordering']
```

Reply:
For some reason with `Meta` class, the inheritance doesn't work and fields from the superclass are missing (only ordering is present).  Fields from `RemoteFilter` come as a dict type, so this feels like a hackish solution:
```
fields = {"url": ["exact", "in", "icontains", "contains"], **RemoteFilter.Meta.fields}
```

Reply:
That is actually way less hackier than you think, and what i think it should be.